models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -76,23 +79,28 @@ aeif_cond_beta_multisynapse::Parameters_::Parameters_()
   , t_ref_( 0.0 )     // ms
   , g_L( 30.0 )       // nS
   , C_m( 281.0 )      // pF
-  , E_ex( 0.0 )       // mV
-  , E_in( -85.0 )     // mV
   , E_L( -70.6 )      // mV
   , Delta_T( 2.0 )    // mV
   , tau_w( 144.0 )    // ms
   , a( 4.0 )          // nS
   , b( 80.5 )         // pA
   , V_th( -50.4 )     // mV
-  , I_e( 0.0 )        // pA
+  , num_of_receptors( 1 )
```

Comment:
I wonder if we should have this as a member. Since it will always be the same as `E_ref.size()`, I think we should should just use
```C++
inline size_t n_receptors() { return E_rev.size(); }
```